mailbox/backup/src/test/java/org/apache/james/mailbox/backup/WithZipHeaderTest.java

```diff
@@ -0,0 +1,48 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.mailbox.backup;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+public class WithZipHeaderTest {
+    private static final short al = 0x6C61;
+    private static final short aq = 0x7161;
+
+    @Test
+    void toLittleEndianShouldReturnLittleEndianRepresentationOfStringAl() {
+        Assert.assertEquals(WithZipHeader.toLittleEndian("al"), (al));
```

Comment:
we are using assertj assertThat for our assertions.
It allows to not mismatch the order of the assertion as you did ;)